ext/spl/spl_iterators.stub.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+function iterator_apply(Traversable $iterator, callable $function, ?array $args = null): int {}
+
+function iterator_count(Traversable $iterator): int {}
+
+function iterator_to_array(Traversable $iterator, bool $use_keys = true): array {}
```

Comment:
I think this one is fine to keep in `php_spl.stub.php`, as that's where it will actually be used. What I'm mainly interested in is that there's a 1-to-1 mapping between the stub file and the using C file, at least for the cases where this is not too hard to achieve.